### PR TITLE
Adding pyramid_log -- noarch, should be simple :-)

### PR DESCRIPTION
**Pull request by ChrisBarker-NOAA**
@conda-forge/help-python 

- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

I asked him, and he said yes. --not sure where he should post a comment.

@daf: maybe post a comment on this PR?

### CODE REVIEW

**Comment by ChrisBarker-NOAA** [comment]
Darn --failed on Windows. I"m not in a position to test there right now -- but if anyone has any ideas, I'm all for it.

**Comment by npavlovikj** [comment]
@ChrisBarker-NOAA , as long as Windows build passes on Azure, we can ignore the AppVeyor failure. The recipe looks good to me - just please remove all redundant comments you have.

File: recipes/pyramid_log/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  imports:
+    - pyramid_log
+
+about:
+  home: https://pypi.org/project/pyramid-log/
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Python logging formatter for Pyramid'
+
```
Comment (npavlovikj) [inline recipes/pyramid_log/meta.yaml]: Please remove all comments and empty line here.

Comment (PythonCHB) [reply recipes/pyramid_log/meta.yaml]: Cool -- nice to know, I've been having a lot of pain with Appveyor!

Stay tuned for a cleaned up meta.yaml

**Comment by PythonCHB** [comment]
Darn! I thought this was all good, and all I changed was removing comments from the meta.yaml :-(

no time to dig deeper now.

**Comment by ChrisBarker-NOAA** [comment]
@npavlovikj: I think it's good to go now !

File: recipes/pyramid_log/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 165ce9460548b09061db6023b5079eb6290d9490a2b4cf28946e3515185d0a52
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (npavlovikj) [inline recipes/pyramid_log/meta.yaml]: Please remove the `build` section - when not using compilers, you put the dependencies in `host`.

Comment (ChrisBarker-NOAA) [reply recipes/pyramid_log/meta.yaml]: I was wondering what the difference was.

Comment (npavlovikj) [reply recipes/pyramid_log/meta.yaml]: You can find better explanation here, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements-section.

File: recipes/pyramid_log/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - pyramid_log
+
+about:
+  home: https://pypi.org/project/pyramid-log/
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE.txt
```
Comment (npavlovikj) [inline recipes/pyramid_log/meta.yaml]: Looks like the license file is already package in the pypi tarball:
```
pyramid_log-0.2.1/
pyramid_log-0.2.1/MANIFEST.in
pyramid_log-0.2.1/setup.py
pyramid_log-0.2.1/README.rst
pyramid_log-0.2.1/LICENSE
```
so, please remove the file LICENSE.txt, and rename the license_file to LICENSE.

Comment (ChrisBarker-NOAA) [reply recipes/pyramid_log/meta.yaml]: good catch, thanks!

**Comment by npavlovikj** [comment]
Thanks for contributing, @ChrisBarker-NOAA 🎉

**Pull request merged**